### PR TITLE
specified units on response values in bones or DC

### PR DESCRIPTION
**Pull request by samgutentag**
fix for helium/blockchain-http#434

### CODE REVIEW

File: docs/api/blockchain/introduction.mdx [hunk 1]
```diff
@@ -53,19 +52,26 @@ next page:
...
+Unless otherwise specified, API calls return values in the following units:
 
-The conversion rate is `100,000,000 bone` per `HNT`.
+| value        | Units        | Conversion to HNT             |
+| ------------ | ------------ | ----------------------------- |
+| `amount`     | bones        | `100,000,000 bone` to `1 HNT` |
+| `balance`    | bones        | `100,000,000 bone` to `1 HNT` |
+| `stake`      | bones        | `100,000,000 bone` to `1 HNT` |
+| `sum`        | bones        | `100,000,000 bone` to `1 HNT` |
+| `fee`        | Data Credits | `100,000 DC` to `1 HNT`       |
```
Comment (cokes518) [inline docs/api/blockchain/introduction.mdx]: this doesn't seem right. The price of Data Credits is fixed in USD (1 Data Credit = $0.00001). so 100,000 DC = $1 USD

File: docs/api/blockchain/introduction.mdx [hunk 2]
```diff
@@ -53,19 +52,26 @@ next page:
...
+| `dc_balance` | Data Credits | `100,000 DC` to `1 HNT`       |
```
Comment (cokes518) [inline docs/api/blockchain/introduction.mdx]: same thing here

File: docs/api/blockchain/validators.mdx [hunk 3]
```diff
@@ -6594,7 +6594,7 @@ GET https://api.helium.io/v1/validators/stats
 
 ```
 
-Returns stats for validators
+Returns stats for validators.
```
Comment (cokes518) [inline docs/api/blockchain/validators.mdx]:
```suggestion
Returns stats for Validators.
```

File: docs/api/blockchain/validators.mdx [hunk 4]
```diff
@@ -7584,6 +7590,12 @@ _Query Parameters_
 
 Rewards for a validator in the last 24 hours: `min_time=-24%20hour`
```
Comment (cokes518) [inline docs/api/blockchain/validators.mdx]:
```suggestion
Rewards for a Validator in the last 24 hours: `min_time=-24%20hour`
```

**Comment by samgutentag** [comment]
DC to USD conversion corrections applied

**Comment by samgutentag** [comment]
closing PR and incorporating changes in broader docs cleanup efforts

**Pull request closed without merging**